internal/exportimport/import.go

```diff
@@ -51,6 +56,17 @@ type SignatureAndKey struct {
 }
 
 func (s *Server) ImportExportFile(ctx context.Context, ir *ImportRequest) (*ImportResposne, error) {
+	// Special case - previous versions may have inserted the filename root as a file.
+	// If we find that, skip attempted processing and just mark as successful.
+	logging.FromContext(ctx).Debugw("does it float", "exportRoot", ir.exportImport.ExportRoot, "zipFilename", ir.file.ZipFilename)
```

Comment:
Not sure if I understand `"does it float"`

Reply:
debugging statement - removed in second commit.